verifier/src/x509.rs

```diff
@@ -6,5 +6,177 @@ mod certs;
 mod crl;
```

Comment:
I think we had discussed not worrying about the other x509 files, but I think at this point it's time to remove them to avoid ambiguity with what parts do what.

Reply:
okay, sounds good